docs/user_guide/recording_and_loading_data.md

```diff
@@ -0,0 +1,128 @@
+# Recording and Loading Data From H2I Simulations
+Detailed data from H2Integrate simulations can be saved and loaded later. This section covers:
+1. [How to record data from a simulation](setting-recorder-parameters)
+2. [How to load data and access recorded data](loading-recorder-files)
+
+(setting-recorder-parameters)=
+## Setting Recorder Parameters
+
+Recording data from a simulation can be enabled in the `driver_config` file with the 'recorder' section. The most simple example is shown below:
+
+```yaml
+name: "driver_config"
+description: "example driver config to show recording options"
+
+general:
+  folder_output: outputs #folder to save recorded data to
+
+recorder:
+  flag: True #set to True to record data
+  file: "cases.sql" #this is the name of the file to record data to.
+```
+In the above example, data will be saved to the file `outputs/cases.sql`.
+
+The file below would have the same behavior as the above example, but highlights the default behavior:
+```yaml
+name: "driver_config"
+description: "example driver config to show recording options"
+
+general:
+  folder_output: outputs #folder to save recorded data to
+
+recorder:
+  flag: True #set to True to record data
+  file: "cases.sql" #this is the name of the file to record data to.
+  overwrite_recorder: False #create a unique recorder for each simulation
+  recorder_attachment: "model" # "driver" or "model"
+  includes: ["*"] # include everything
+  excludes: ["*resource_data"] # don't include resource data
+```
+
+- **overwrite_recorder**: If False or not specified, H2I will make a new filename for the recorder that doesn't yet exist in the `outputs` folder. If `cases.sql` exists, it will make a new file named `cases0.sql`. If `cases.sql` and `cases0.sql` exist, it will make a new file named `cases1.sql`, etc. If set to True, it will overwrite an existing `cases.sql` file from previous runs.
+- **recorder_attachment**: Must be `model` or `driver`, defaults to `model`. We recommend attaching the recorder to the driver if running an optimization or design of experiments in parallel.
```

Comment:
Why do we recommend attaching to the driver? Might not be relevant info but this was my first though after reading this.

Reply:
Good call, it's due to a limitation at the OpenMDAO level -- I updated the docs accordingly.